kiwi/repository/yum.py

```diff
@@ -248,9 +258,9 @@ def _create_runtime_config_parser(self):
         self.runtime_yum_config.set(
             'main', 'gpgcheck', self.gpg_check
         )
-        self.runtime_yum_config.set(
-            'main', 'repo_gpgcheck', self.gpg_check
-        )
+        # We are not setting repo_gpgcheck, which forces repository
+        # signature checks, because I could not manage to make it work.
+        # Nevertheless, package signature validation is functional.
```

Comment:
Yes CentOS repos are signed, this is true. But I just could not make it run on my openSUSE host (tested with leap 42.2 and tumbleweed clean images). I did a custom `yum_config` file, as simple as:
```
[main]
cachedir = /tmp/yum-cache
reposdir = /tmp/yum-repos
pluginconfpath = /tmp/yum-pluginconf
debuglevel = 10
repo_gpgcheck = 1
metadata_expire = 180
```
Added repo in `/tmp/yum-repos/OSS.repo`:
```
[OSS]
name = OSS
baseurl = http://mirror.centos.org/centos/7/os/x86_64/
```
and then executed:

```
$ rpm --import http://mirror.centos.org/centos/7/os/x86_64/RPM-GPG-KEY-CentOS-7
$ yum -c yum_config makecache
```
Also tried some other similar combinations, always with the same result:
```
Config time: 0.008
Yum Version: 3.4.3
COMMAND: yum -c yum_config makecache 
Installroot: /
OSS/signature                                                                                                                                                                                                               |  811 B     00:00     
OSS/signature                                                                                                                                                                                                               | 3.6 kB     00:00 ... 
http://mirror.centos.org/centos/7/os/x86_64/repodata/repomd.xml: [Errno -1] Gpg Keys not imported, cannot verify repomd.xml for repo OSS
Trying other mirror.
Error: Cannot retrieve repository metadata (repomd.xml) for repository: OSS. Please verify its path and try again
```